### ISSUE TITLE
Which version of React-Native do we want to support ?

### ISSUE THREAD

**Issue by arribbar** [leecade/react-native-swiper#499]
Hi everyone,

I was wondering which version of RN do we want to support. 
With the last changes, we support RN >= 0.46.
Do we want to support previous versions ? If yes, from which version?

**Comment by mrdezzods** [comment]
Is there anything specific that we want to implement but limited by the RN version? If not, I would keep support for as older version as possible.

**Comment by arribbar** [comment]
I think for now there is only ViewPropTypes which need to be changed. I'll accept https://github.com/leecade/react-native-swiper/pull/498 to do so

**Comment by mrdezzods** [comment]
Indeed. I requested a slight change on that. After that we can merge it.
